drivers/ethernet/phy/Kconfig

```diff
@@ -91,4 +91,20 @@ config PHY_MONITOR_PERIOD
 	  periodically executed to detect and report any changes in the
 	  PHY link status to the operating system.
 
+config PHY_OA_TC14_PLCA_LIB
+	bool "Open Alliance TC14 PLCA generic lib"
+	default n
```

Comment:
The options are "off" by default so `default n` can be omitted.

Reply:
Ok I removed `default n` and updated the PR. Thanks for the comment.